OutOfSchool/OutOfSchool.Migrations/Program.cs

```diff
@@ -0,0 +1,93 @@
+﻿using IdentityServer4.EntityFramework.DbContexts;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using MySqlConnector;
+using OutOfSchool.Common;
+using OutOfSchool.Common.Config;
+using OutOfSchool.Common.Extensions;
+using OutOfSchool.Common.Extensions.Startup;
+using OutOfSchool.IdentityServer.KeyManagement;
+using OutOfSchool.Services;
+using OutOfSchool.Services.Models;
+
+var host = Host.CreateDefaultBuilder(args)
+    .ConfigureServices((context, services) =>
+    {
+        var config = context.Configuration;
+        // TODO: Move version check into an extension to reuse code across apps
+        var mySQLServerVersion = config["MySQLServerVersion"];
+        var serverVersion = new MySqlServerVersion(new Version(mySQLServerVersion));
+        if (serverVersion.Version.Major < Constants.MySQLServerMinimalMajorVersion)
+        {
+            throw new Exception("MySQL Server version should be 8 or higher.");
+        }
+
+        var connectionString = config.GetMySqlConnectionString<MySqlGuidConnectionOptions>(
+            "DefaultConnection",
+            options => new MySqlConnectionStringBuilder
+            {
+                Server = options.Server,
+                Port = options.Port,
+                UserID = options.UserId,
+                Password = options.Password,
+                Database = options.Database,
+                GuidFormat = options.GuidFormat.ToEnum(MySqlGuidFormat.Default),
+            });
+
+        var migrationsAssembly = "OutOfSchool.Migrations";
```

Comment:
just in case, for less hardcoding we can copy old thing from IdentityServer like `var migrationsAssembly = typeof(Program).GetTypeInfo().Assembly.GetName().Name;`

Reply:
Done